Beam/Include/Beam/Parsers/Types.hpp

```diff
@@ -23,43 +23,43 @@ namespace Beam::Parsers {
 
   /** Specifies the default parser to use for a given type. */
   template<typename T, typename = void>
-  const auto default_parser = std::enable_if_t<!std::is_same_v<T, T>>();
+  static const auto default_parser = std::enable_if_t<!std::is_same_v<T, T>>();
```

Comment:
This is worth changing but mark these as ```inline``` instead of ```static```. I don't want every translation unit to have its own independent definition of these.